### PR TITLE
(#2872, #2884) Add automated regression tests for v1.2.1

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

- Add test package and test for functionality of `[A] Yes to all` prompt actions
- Add test for functionality of `choco upgrade` when given multiple packages to upgrade at once.

## Motivation and Context

Keepin' up with the tests.

## Testing

1. Run vagrant image tests for `choco install` and `choco upgrade` command
2. New tests should pass on build from `hotfix/1.2.1` but fail under choco `1.2.0`.

### Operating Systems Testing

N/A

## Change Types Made

* [x] Tests added
* [ ] Bug fix (non-breaking change).
* [ ] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [ ] Documentation changes.
* [ ] PowerShell code changes.

## Change Checklist

* [ ] Requires a change to the documentation.
* [ ] Documentation has been updated.
* [x] Tests to cover my changes, have been added.
* [x] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v2 compatibility checked?

## Related Issue

Tests for #2872

### CODE REVIEW

**Review by corbob — COMMENTED** [review]
These look good to me. I've run the tests against Test Kitchen, and aside from the multiple packages failing due to missing the package on the internal repository they all worked as expected.

Just a note about possibly not `Trim()`ing strings we've already `Trim()`ed 🤷

File: tests/chocolatey-tests/commands/choco-install.Tests.ps1 [hunk 1]
```diff
@@ -1594,4 +1594,34 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
...
+
+        It "Installed the packages to the lib directory" {
+            $PackageUnderTest | ForEach-Object {
+                "$env:ChocolateyInstall\lib\$_" | Should -Exist
+            }
+        }
+
+        It "Ran both installation scripts after selecting [A] Yes to all at the first prompt" {
+            $promptLine = "Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint):"
+            $prompts = $Output.Lines | Where-Object { $_.Trim() -eq $promptLine }
```
[the remark this answers is not in the document]
Comment (vexx32) [reply tests/chocolatey-tests/commands/choco-install.Tests.ps1:1622]: Ah we do? fantastic, I'll fix that up.

[the remark this answers is not in the document]
Comment (vexx32) [reply tests/chocolatey-tests/commands/choco-install.Tests.ps1]: All sorted, rebased as well. Thanks!

**Review by corbob — APPROVED** [review]
LGTM!

**Comment by corbob** [comment]
Thanks for getting these tests added @vexx32!

**Pull request merged**